### PR TITLE
ISPN-12588 Fix server's common.sh to handle custom paths correctly

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-12588

### CODE REVIEW

**Review by karesti — APPROVED** [review]
code LGTM Tristan but I did not test it locally myself

**Comment by tristantarrant** [comment]
Wolf is testing this. Let's wait for his response

File: server/runtime/src/main/server/bin/cli.sh [hunk 1]
```diff
@@ -122,16 +127,16 @@ if $linux; then
     for var in $CONSOLIDATED_OPTS
     do
        # Remove quotes
-       p=$(echo $var | tr -d "'")
+       p=$(echo "$var" | tr -d "'")
        case $p in
-         -Dinfinispan.server.base.dir=*)
```
Comment (wfink) [inline server/runtime/src/main/server/bin/cli.sh:127]: If not used I would remove ispn.server.*.path use

File: server/runtime/src/main/server/bin/cli.sh [hunk 2]
```diff
@@ -167,65 +172,65 @@ if $darwin || $freebsd || $other ; then
...
 if [ "x$ISPN_CONFIG_DIR" = "x" ]; then
-   ISPN_CONFIG_DIR="$ISPN_BASE_DIR/conf"
+   ISPN_CONFIG_DIR="$ISPN_ROOT_DIR/conf"
 fi
 
 # For Cygwin, switch paths to Windows format before running java
 if $cygwin; then
     ISPN_HOME=$(cygpath --path --windows "$ISPN_HOME")
     JAVA_HOME=$(cygpath --path --windows "$JAVA_HOME")
     ISPN_MODULEPATH=$(cygpath --path --windows "$ISPN_MODULEPATH")
```
Comment (wfink) [inline server/runtime/src/main/server/bin/cli.sh:212]: MODULEPATH seems a leftover

**Review by wfink — COMMENTED** [review]
CLI change works for me.
The comments are only to remove unnecessary

**Review by wfink — APPROVED** [review]
common.sh would now use infinispan.server.*.path if set directly
but the -P <property.file> is not used if set there.
We should document or add this

File: server/runtime/src/main/server/bin/cli.sh (hunk 1 above)
Comment (tristantarrant) [reply server/runtime/src/main/server/bin/cli.sh:127]: There are some commands that reference those paths: `user`, `credential`, so I'd rather leave those in

**Review by wfink — APPROVED** [review]
Works with direct -Dinfinispan.server.*.path settings or -P properties file

**Pull request merged**